### PR TITLE
New sites added

### PR DESCRIPTION
**Pull request by falconcode16**

### CODE REVIEW

**Comment by pkubiak** [comment]
Hi, your probably forgotten to upload updated `data.json` file.

File: sherlock/resources/data.json [hunk 1]
```diff
@@ -2213,6 +2213,55 @@
     "url": "http://uid.me/{}",
     "urlMain": "https://uid.me/",
     "username_claimed": "blue",
-    "username_unclaimed": "noonewouldeverusethis7"
+    "username_unclaimed": "noonewouldeveethis7"
+  },
+  "TensorFlow": {
+    "errorType": "status_code",
+    "url": "https://www.tensorflow.org/",
+    "urlMain": "https://www.tensorflow.org/,,
```
Comment (pkubiak) [inline sherlock/resources/data.json:2221]: Type, you miss `"`

File: sherlock/resources/data.json [hunk 2]
```diff
@@ -2213,6 +2213,55 @@
     "url": "http://uid.me/{}",
     "urlMain": "https://uid.me/",
     "username_claimed": "blue",
-    "username_unclaimed": "noonewouldeverusethis7"
+    "username_unclaimed": "noonewouldeveethis7"
+  },
+  "TensorFlow": {
+    "errorType": "status_code",
+    "url": "https://www.tensorflow.org/",
```
Comment (pkubiak) [inline sherlock/resources/data.json:2220]: Your urls miss `{}` parts used for username substitution. Take a look at https://github.com/sherlock-project/sherlock/wiki/Adding-Sites-To-Sherlock.

You can also run tests for your sites locally with:
```bash
> python3 -m unittest tests.all --verbose
```

**Comment by sdushantha** [comment]
Thank you for putting time and effort into this PR, but I'm not so keen on merging this PR because of the commit messages. They dont tell other people what that commit does other than it was a file upload. 

I'm closing this PR. Create a new one and this time use `git` to upload the files so that the commit history makes sense.

**Pull request closed without merging**